tweet_recommendations/rank_pipeline.py

```diff
@@ -0,0 +1,28 @@
+from tweet_recommendations.embeddings.fast_text import get_fasttext_tweet_embedding
+from tweet_recommendations.embeddings.word2vec import get_w2v_tweet_embedding
+from tweet_recommendations.utils.clients import get_wcrft2_results_for_text
+
+sample_tweet_content = (
+    "Przykładowa treść tweeta do której mają być zasugerowane hasztagi."
+)
+use_w2v_embedding = True
+
+
+def get_hashtag_rank_for_given_tweet_text(tweet: str):
+    tagged_tweet_content = get_wcrft2_results_for_text(tweet)
+
+    embeddings = (
+        get_w2v_tweet_embedding(tagged_tweet_content)
+        if use_w2v_embedding
+        else get_fasttext_tweet_embedding(tagged_tweet_content)
+    )
+
+    hashtag_rank = []
+    # hashtag_rank = perform_some_graph_rank_magic(embeddings)
+
+    return hashtag_rank
```

Comment:
Czemu jest zwracana zawsze pusta lista i po co ten komentarz wyżej?

Reply:
Jest to jeszcze nie zaimplementowane - tzn potrzeba napisać jakąś funkcję, która zwróci ranking hasztagów. Na ten moment tego jeszcze nie ma. komentarz jest tylko napisany po to, żeby wiedzieć, gdzie to wpiąć jak już będzie zrobione.

Reply:
To można zamiast tego
```python
def jakas_sensowna_nazwa_ktora_mozna_juz_teraz_ustalic(embeddings):
    raise NotImplementedError
.
.
.
    hashtags = jakas_sensowna_nazwa_ktora_mozna_juz_teraz_ustalic(embeddings)
    return hashtags
```